src/main/java/uk/gov/hmcts/reform/civil/enums/CaseState.java

```diff
@@ -13,7 +13,7 @@ public enum CaseState {
     AWAITING_RESPONDENT_ACKNOWLEDGEMENT("Awaiting respondent acknowledgement"),
     CASE_DISMISSED("Case Dismissed"),
     AWAITING_APPLICANT_INTENTION("Awaiting applicant intention"),
-    PROCEEDS_IN_HERITAGE_SYSTEM("Proceeds in heritage system"),
+    PROCEEDS_IN_HERITAGE("Proceeds in heritage system"),
```

Comment:
you should add a new entry rather than modify the existing parent case state which is proceeds_in_heritage_system

Reply:
Added Proceeds_in_heritage_system back in the code to maintain the existing parent case state.